openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -3202,7 +3202,7 @@ interface UtilDAO {
                 + "(SELECT COUNT(*) FROM dashboard_entity <cond>) as dashboardCount, "
                 + "(SELECT COUNT(*) FROM pipeline_entity <cond>) as pipelineCount, "
                 + "(SELECT COUNT(*) FROM ml_model_entity <cond>) as mlmodelCount, "
-                + "(SELECT (SELECT COUNT(*) FROM database_entity <cond>) + "
```

Comment:
can we add a new query , since we use this for listing database_entities !

Reply:
Resolved offline.